### DISCUSSION TITLE
Where do you generally install cutlass?

### DISCUSSION THREAD

**Discussion by Enchante503** [NVIDIA/cutlass#1712] [General]
I'm using WSL2 (Ubuntu).

Where do you generally install cutlass?

I asked ChatGPT and they recommended usr/local/.
There are folders like cuda-11.8, cuda-12., so that may be correct.

Would it be better to name the folder with an easy-to-understand version, like "cuda-11.8"?
Is it okay to just call it "cutlass"?

**Comment by thakkarV** [comment]
I'd recommend just fetching it using cmake fetch content or a git submodule in your project for under a third party dir.

**Reply by Enchante503** [reply]
Hello thakkarV
Thank you very much. I'll try using git submodule.
As a beginner, I learned something new.
